uber/models.py

```diff
@@ -1182,14 +1182,14 @@ def last_first(cls):
     @property
     def watchlist_guess(self):
         try:
-            with sa.Session() as session:
-                return session.guess_attendee_watchentry(self)
+            with Session() as session:
+                return [u.__dict__ for u in session.guess_attendee_watchentry(self)]
```

Comment:
Dunno what to tell you, but I changed ``.__dict__`` to ``.to_dict()`` and it worked:

```python
￼￼>>> Attendee(first_name='Some', last_name='Jerk').banned
[{'birthdate': datetime.date(1999, 1, 1), 'id': '0c6c6161-76a3-42bc-9a7e-eacf50c66a2d', 'last_name': 'Jerk', 'action': 'stop being a jerk', 'active': True, 'first_names': 'Some', '_model': 'WatchList', 'reason': 'jerkiness', 'email': 'jerky@mailinator.com'}]
```